src/vm/i386/asmhelpers.S

```diff
@@ -472,10 +472,12 @@ NESTED_END CallDescrWorkerInternal, _TEXT
 #ifdef _DEBUG
 // int __fastcall HelperMethodFrameRestoreState(HelperMethodFrame*, struct MachState *)
 LEAF_ENTRY HelperMethodFrameRestoreState, _TEXT
+    #define     RETN 8
```

Comment:
The PAL_STDCPP_COMPAT is only defined when building some debugger stuff. Its goal is to allow building code that depends on PAL and still needs to pull in some platform specific headers, which is forbidden in libcoreclr.so.
I don't know though why the __fastcall is defined as __stdcall here. It looks like some historical thing - it was there from the very beginning of the port. So I believe we should remove the redefinition of the __fastcall at that place and all will be good.